speech.py

```diff
@@ -63,48 +96,126 @@ def connect_wave(self, cb):
     def connect_idle(self, cb):
         self._cb['idle'] = self.connect('idle', cb)
 
+    def set_kokoro_voice(self, voice_name): #TODO - function set here to this can be called for when switching happens via GUI
```

Comment:
The comment here is a bit confusing, you could also use a setter and that would remove the need for the comment.